### ISSUE TITLE
transactions and sessions support

### ISSUE THREAD

**Issue by lagripe** [mongo-dart/mongo_dart#274]
Hey, 

Thanks for your efforts for building the library, will there be transactions/sessions support any time soon ?

Regards.

**Comment by giorgiofran** [comment]
No, unfortunately at present I have not the time to do such a development, even if the feature is a _nice to have_.
